php/blocks/controls/class-control-abstract.php

```diff
@@ -281,6 +275,31 @@ public function render_settings_post_type_rest_slug( $setting, $name, $id ) {
 		<?php
 	}
 
+	/**
+	 * Gets the REST slugs of public post types, other than 'attachment'.
+	 *
+	 * @return array {
+	 *     An associative array of the post type REST slugs.
+	 *
+	 *     @type string $rest_slug The REST slug of the post type.
+	 *     @type string $name The name of the post type.n
+	 * }
+	 */
+	public function get_post_type_rest_slugs() {
```

Comment:
Great idea separating this out into its own method.

Reply:
Thanks, Luke!